src/FieldDescription/FieldDescriptionInterface.php

```diff
@@ -27,7 +27,7 @@
  *  role?: string|string[],
  *  sort_field_mapping?: array<string, mixed>,
  *  sort_parent_association_mappings?: array<array<string, mixed>>,
- *  sortable?: bool,
+ *  sortable?: string|bool,
```

Comment:
🤔 I think this and also the changelog are already in `master`

Reply:
Fixed